client/scripts/views/pages/create_community.ts

```diff
@@ -1016,25 +1016,16 @@ interface CreateCommunityState {
   activeForm: string;
 }
 
-const CreateCommunityModal: m.Component<
+const CreateCommunity: m.Component<
   CreateCommunityAttrs,
   CreateCommunityState
 > = {
-  oncreate: () => {
-    mixpanel.track('New Community', {
-      'Step No': 1,
-      Step: 'Modal Opened',
-    });
-  },
   oninit: (vnode) => {
     vnode.state.activeForm = CommunityType.OffchainCommunity;
   },
   view: (vnode: m.VnodeDOM<CreateCommunityAttrs, CreateCommunityState>) => {
-    return m('.CreateCommunityModal', [
-      m('.compact-modal-title', [
-        m('h3', 'New Commonwealth Community'),
-        m(CompactModalExitButton),
-      ]),
+    return m('div', { class: 'container CreateCommunity' }, [
```

Comment:
One last change before merging this PR: 

- We should return this form inside of a Sublayout component like all of the other pages on CW. Linking to a blank page with just a form isn't going to cut it. 
  - Pass the attr `hideSidebar: true;` into Sublayout as this page isn't scoped within a community, so no need for the sidebar. (This should not impact the view of the SidebarSwitcher, just the Sidebar menu)